src/OrbitBase/GetProcessIdsLinuxTest.cpp

```diff
@@ -62,8 +62,9 @@ TEST(GetProcessIdsLinux, GetTidsOfProcess) {
   }
   thread.join();
 
-  std::vector<pid_t> expected_tids{main_tid, thread_tid};
-  EXPECT_THAT(returned_tids, ::testing::UnorderedElementsAreArray(expected_tids));
+  // There might be more than these two threads (i.e. when compiled with sanitizers enabled)
+  EXPECT_THAT(returned_tids, ::testing::Contains(main_tid));
+  EXPECT_THAT(returned_tids, ::testing::Contains(thread_tid));
```

Comment:
Let me know if you prefer to keep this EXPECT to check for equality. I can certainly also solve this with copybara-ifdefs.

Reply:
As you think it's best, no concern from my side.